rest_framework-stubs/fields.pyi

```diff
@@ -39,6 +39,10 @@ class CreateOnlyDefault(object):
     def set_context(self, serializer_field: Field) -> None: ...
     def __call__(self) -> Any: ...
 
+class CurrentUserDefault(object):
+    def set_context(self, serializer_field: Field) -> None: ...
+    def __call__(self) -> Any: ...
```

Comment:
@mkurnikov are we okay with `Any` in these cases?

Reply:
Sorry, I somehow missed it. Just add some `# TODO:` for later, with explanation why it's difficult to add return type now, and it's good to go.

Reply:
I pushed a change to make it `Union[AnonymousUser, User]` which is what [get_user](https://github.com/typeddjango/django-stubs/blob/335c5765c3960e3ebd296b65ebac27b09b1e44fe/django-stubs/contrib/auth/middleware.pyi#L8) returns. This assumes both `AuthenticationMiddleware` is used and there's no custom user model in use.